src/classes/UTIL_OrgTelemetry_SObject_BATCH_TEST.cls

```diff
@@ -354,91 +365,8 @@ private class UTIL_OrgTelemetry_SObject_BATCH_TEST {
     }
 
     /**
-     * @description Confirms Opportunity Payment metrics are captured in FeatureManagement at the end of the batch
-     * when Auto Payments are enabled
-     */
-    /* TODO: Holding on Data_OppPaymentsWithoutPaymentsEnabled metric, reenable test when
```

Comment:
@andrewyu-salesforce , let's leave these two commented out tests since there is a mention in the comment about re-enabling them at some point.